pandas/compat/__init__.py

```diff
@@ -18,6 +18,7 @@
 PY38 = sys.version_info >= (3, 8)
 PY39 = sys.version_info >= (3, 9)
 PYPY = platform.python_implementation() == "PyPy"
+IS64 = sys.maxsize > 2 ** 32
```

Comment:
I can't imagine it matters in practice but in theory this should be 2 ** 32 - 1

Reply:
I copied this from https://docs.python.org/3/library/platform.html#cross-platform

```python
is_64bits = sys.maxsize > 2**32
```